src/SelectionInfoView.tsx

```diff
@@ -1,34 +1,58 @@
 import * as React from "react";
 
 import { MarkerFileData } from "./DataTypes";
+import { computeAngle} from "./Calculations";
 
 interface Props {
 	markerData: MarkerFileData;
 	selectedMarkers: number[];
 	frame: number;
 }
 
-export default function SelectionInfoView(props: Props) {
-	if(props.selectedMarkers.length === 0) return <p>Nothing selected.</p>;
-	return <div>
-		{
-			props.selectedMarkers.map(idx => {
-				const info = props.markerData.markers[idx];
-				const frameInfo = props.markerData.frames[props.frame].positions[idx];
+	export default function SelectionInfoView(props: Props) {
```

Comment:
indentation is wrong here

Reply:
still relevant, all these lines are too far

Reply:
I see what you mean now. Fixed.